### ISSUE TITLE
JAWS is not reading the change in value of a vertical slider

### ISSUE THREAD

**Issue by aayushdoda** [FreedomScientific/VFO-standards-support#394]
**Summary**
JAWS is not reading the change in value if there is vertical slider.

**Example**:

     1. Open IE
     2. Go to https://www.microsoft.com/en-us/sql-server/sql-server-2017-pricing 
     3. Click on 'Watch now' CTA in HERO to launch One Player video
     4. Move volume slider up and down using the arrow key.
     5. It reads the value only once and after that it stops reading.

**or**
    1.  Open IE
    2. Go to  https://www.w3.org/TR/2016/WD-wai-aria-practices-1.1-20160317/examples/slider/slider-2.html
    3. Move volume slider up and down using the arrow key.
    4. It reads the value only once and after that it stops reading.

**Expected result**
JAWS should read all change in the volume.

**Actual result**
JAWS is reading the change in value only once.

**Comment by JAWS-test** [comment]
Duplicate of https://github.com/FreedomScientific/VFO-standards-support/issues/219

**Comment by JeremiahRogersVispero** [comment]
Confirmed to be working properly using JAWS 2026.2606.132 with [this sample,](https://cdpn.io/pen/debug/oNmpoYg) provided by @stevefaulkner, for #219. Thanks for identifying this duplicate, @JAWS-test.
